docs/Reference/Multikey-Parameters.md

```diff
@@ -0,0 +1,95 @@
+---
+description: TOML file parameters for multi signers.
+---
+
+# Multikey Parameters
+
+Create a TOML configuration file for each key that signs transactions. Keys can be stored in a V3 keystore file, Hashicorp Vault, or Azure Key Vault.
+
+The filename must be formatted as `[<prefix>]<accountAddress>.toml` and must be located in the directory 
+specified using the [`multikey-signer --directory`](../CLI/CLI-Syntax#multikey-options) command line option.
+
+The optional `[metadata]` section in the TOML files is used for data that is not required by EthSigner.
```

Comment:
_is used for_ -> _contains_ 

Suggested edit to reduce number of words

Reply:
Done